tutorials/deploy-mdbooks-serverless-containers/index.mdx

```diff
@@ -0,0 +1,114 @@
+---
+meta:
+  title: Fast deployment of documentation on Serverless Containers with mdbooks
+  description: Step-by-step guide to deploy mdbooks on Scaleway Serverless Containers.
+content:
+  h1: Fast deployment of documentation on Serverless Containers with mdbooks
+  paragraph: Step-by-step guide to deploy mdbooks on Scaleway Serverless Containers.
+tags: docker container deploy serverless
+hero: assets/scaleway-umami.webp
+categories:
+  - containers
+  - container-registry
+dates:
+  validation:
+  posted:
+---
+
+Publish modern online books, for product, API documentation, tutorials, course material or anything that requires a clean, easily navigable and customizable presentation.
+
+This tutorial uses `mdbooks` to publish simple documentation but the main goal of this tutorial is to show how simple it is to
+pick a project, package it inside an image and deploy it on Serverless Containers.
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- Docker installed on your local machine
+
+## Why deploying on Serverless
+
+Serverless products are perfect for cost efficiency with a pay-as-you-go model and scale very well.
+
+With zero infrastructure management and many tools to integrate in CI/CD environments, `mdbooks` can benefit a lot of Serverless Containers.
+
+## Local setup
+
+1. Follow the [installation instructions](https://rust-lang.github.io/mdBook/guide/installation.html) of mdbooks.
+
+2. Run commands:
+- `mdbook init my-first-book` in order to create a sample book.
+- `cd my-first-book` to open the directory created by init command.
+
+3. *(optional)* Now you can edit the content of the book to publish.
+
+4. It is recommended to test the book using the command: `mdbook test`.
+
+## Prepare Container Registry
+
+<Message type="note">
+  We recommend using Scaleway Container Registry instead of external registries to avoid errors of rate limiting and risks regarding evolutions of CGU and pricing.
+</Message>
+
+1. Create a file named `Dockerfile` in this folder containing the following code:
+
+```dockerfile
```

Comment:
This whole code block needs to be indented two levels, to avoid breaking numbered steps